docs/source/alignment.ipynb

```diff
@@ -0,0 +1,647 @@
+{
```

Comment:
<p>have you done this plot without detrending? Looks different. I dont say it is wrong.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bradyrx/climpred/pull/338/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>